impl/client/src/main/resources-filtered/web/common-ui-require-js-cfg.js

```diff
@@ -290,6 +290,12 @@
       "pentaho/visual/DefaultView": {module: "pentaho/ccc/visual/BarNormalizedHorizontal"}
     }
   };
+  requireModules["pentaho/visual/models/Waterfall"] = {
```

Comment:
RequireJS changes affecting vizs must be repeated in CGG: https://github.com/webdetails/cgg/blob/6d7aca9eadd1a64d27588c832a1e97c59f43d72e/core/src/main/javascript/pt/webdetails/cgg/resources/define-cfg.js.

Reply:
Oh, that's to support the exporting/printing of this visualization with CGG, which is used by Analyzer for the export operation. IIRC, CDF/CDE dashboards can also use CGG to export visualizations. Unfortunately, CGG does not load the full Pentaho webcontext.js as the browser does, requiring the RequireJS configurations to be repeated there.